Templates/Clean Swift/SceneSupport.xctemplate/AlertPresentable.swift

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
just have these files in project, remove them from templates, you use scene template to create a scene, not the whole project

which means that these temoplaytes would work only on a well-prepared project, but htat's fine